packages/date-picker/test/dropdown.test.js

```diff
@@ -165,43 +196,3 @@ describe('dropdown', () => {
     });
   });
 });
-
-class DatePickerWrapper extends PolymerElement {
-  static get template() {
-    return html`<vaadin-date-picker id="datepicker" label="foo" clear-button-visible></vaadin-date-picker>`;
-  }
-}
-
-customElements.define('vaadin-date-picker-wrapper', DatePickerWrapper);
-
-describe('wrapped', () => {
-  let datepicker, clearButton, toggleButton;
-
-  beforeEach(() => {
-    datepicker = fixtureSync(`<vaadin-date-picker-wrapper></vaadin-date-picker-wrapper>`).$.datepicker;
-    toggleButton = datepicker.shadowRoot.querySelector('[part="toggle-button"]');
-    clearButton = datepicker.shadowRoot.querySelector('[part="clear-button"]');
-  });
-
-  it('should not close on calendar icon down', (done) => {
```

Comment:
This test is removed without a replacement. It was added in https://github.com/vaadin/vaadin-date-picker/pull/177 to workaround an issue with `iron-dropdown`, and the relevant method was removed when switching to `vaadin-overlay` in https://github.com/vaadin/vaadin-date-picker/pull/482.